.github/workflows/cicd_lambda_metrics_cleaner.yml

```diff
@@ -0,0 +1,67 @@
+name: CI/CD Lambda Metric Cleaner
+
+on:
+  push:
+    paths:
+    - 'cmd/metrics/**'
+    - 'Makefile'
+    - '.github/workflows/cicd_lambda_metrics_cleaner.yml'
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-18.04
+    steps:
+    - name: Set up Go 1.13
+      uses: actions/setup-go@v1
+      with:
+        go-version: 1.13
+      id: go
+
+    - uses: actions/checkout@v2
+
+    - name: Get dependencies
+      run: |
+        go get -v -d ./...
+        if [ -f Gopkg.toml ]; then
+            curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
+            dep ensure
+        fi
+
+    - name: Build the metrics cleaner lamdba
+      run: make build-metrics-lambda
+    
+    - name: Zip the metrics cleaner code
+      run: |
+        cp bin/metrics bin/main
+        zip -j bin/metrics.zip bin/main
+    
+    - name: Upload artifact (only on master)
+      if: github.ref == 'refs/heads/master'
+      uses: actions/upload-artifact@v1
+      with:
+        name: metrics
+        path: bin/metrics.zip
+
+  deploy:
+    name: Deploy
+    runs-on: ubuntu-18.04
+    needs: [build]
+    if: github.ref == 'refs/heads/master'
+    steps:
+    - name: Configure AWS cli
+      uses: aws-actions/configure-aws-credentials@v1
+      with:
+        aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        aws-region: eu-west-1
+
+    - name: Download artifact
+      uses: actions/download-artifact@v1
+      with:
+        name: metrics
+
+    - name: Deploy and update lambda
+      run: |
+        aws s3 cp metrics/metrics.zip s3://tzlink-metrics-lambda-dev/v1.0.0/metrics.zip
+        aws lambda update-function-code --function-name metrics --s3-bucket tzlink-metrics-lambda-dev --s3-key v1.0.0/metrics.zip
```

Comment:
Hum. Currently, a version control of the name of the archive is complicated because there is no true versionning of the lambda (except the internal one made by Lambda locally). So I'm not sure that we could make it :/